WikiDrips/UIImageExtensions.swift

```diff
@@ -10,7 +10,8 @@ import UIKit
 
 extension UIImage {
     
-    static func image(withInitials initials: String?, in bounds: CGRect) -> UIImage {
+    static func image(withInitials initials: String?) -> UIImage {
+        let bounds = CGRect(x:0,y:0,width:32,height:32)
```

Comment:
Did you want to change this to:
`static func image(withInitials initials: String?, size: CGSize) -> UIImage` so this image extension could be more reusable in other situations?